app/mapping.py

```diff
@@ -16,25 +19,38 @@ class MappingNotFound(BadRequest):
 
 
 class Mapping(object):
+    filter_field_prefix = "dmfilter"
```

Comment:
Might these be better defined as constants? Like we do with `SERVICES_MAPPING_FILE_SPEC ` and `SERVICE_ID_HASH_FIELD_NAME `.

Reply:
The vague idea being that it would be possible to subclass `Mapping` with a different set of prefixes.